vendor/k8s.io/kubernetes/pkg/kubelet/kubelet_server_journal.go

```diff
@@ -135,6 +135,16 @@ func (a *journalArgs) Args() []string {
 // args to the provided writer, timing out at a.Timeout. If an error occurs a line
 // is written to the output.
 func (a *journalArgs) Copy(w io.Writer) {
+	// determine whether we have more than a single boot record, and if so, include the previous
+	// boot so that we always have logs from before a restart
+	args := a.Args()
+	args = append(args, "--boot")
+	if out, err := exec.Command("journalctl", "--list-boot").CombinedOutput(); err == nil && bytes.Count(bytes.TrimSpace(out), []byte("\n")) > 1 {
+		args = append(args, "-1")
```

Comment:
When we have a previous boot, we want to call `journalctl` with both `--boot -1` and `--boot -0`.  My reading of your patch (possibly to narrowly) is that we'd only call it once?